src/groovy/au/org/emii/aatams/JodaOverrides.groovy

```diff
@@ -0,0 +1,42 @@
+package au.org.emii.aatams
+
+import org.joda.time.Interval
+import org.joda.time.base.AbstractInterval
+
+class JodaOverrides {
+
+    // Would be nicer to do this with an "Extension Module", but alas, that's
+    // only available in groovy 2.0.
+    //
+    // An extension module would avoid the need to mock/unmock in every test
+    // which executes the overlapsAny code.
+    static void apply() {
+        AbstractInterval.metaClass.contains = { List instantList ->
```

Comment:
The method is overloaded, but in those cases it's always working on one of something, so there's no ambiguity.

It's up to you but my preference would be to remove the ambiguity.